gremlin-go/driver/connection_test.go

```diff
@@ -208,4 +201,30 @@ func TestConnection(t *testing.T) {
 			assert.Nil(t, err)
```

Comment:
the test on line 199 fails with this message with these changes:
```Error:      	Not equal: 
            	expected: "[0]"
            	actual  : "0"```
Can I change this to 0?